### PR TITLE
NOJIRA: Add committers.md

### PR DESCRIPTION
**Pull request by bjagg**
##### Checklist

-   [x] the [individual contributor license agreement][] is signed
-   [x] commit message follows [commit guidelines][]
-   [x] documentation is changed or added

##### Description of change

[individual contributor license agreement]: https://github.com/Jasig/uPortal/blob/master/CONTRIBUTING.md#individual-contributor-license-agreement
[commit guidelines]: https://github.com/Jasig/uPortal/blob/master/CONTRIBUTING.md#commit
[message properties]: https://github.com/Jasig/uPortal/tree/master/uportal-war/src/main/resources/properties/i18n
[WCAG 2.0 AA]: https://www.w3.org/WAI/WCAG20/quickref/?levels=aaa&technologies=smil%2Cpdf%2Cflash%2Csl

### CODE REVIEW

File: docs/committers.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
+# Committers
```
Comment (ChristianMurphy) [inline docs/committers.md]: Would the headers be clearer with a blank newline after?

**Comment by blamonet** [comment]
@bjagg a thought, we might want to make the md file name in caps to raise this document to the top of the directory.

File: docs/committers.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+ - Drew Wills
+ - James Wennmacher
+ - Jeff Cross
+ - Josh Helmer
+ - Misagh Moayyed
+ - Nicholas Blair
+ - Paul Spaude
+ - Tim Levett
+ - Timothy A Vertein
+ - William G. Thompson, Jr.
```
Comment (ChristianMurphy) [inline docs/committers.md]: would if be helpful to include github id in parenthesis?

**Comment by ChristianMurphy** [comment]
> make the md file name in caps

+1 :+1: 

> this document to the top of the directory.

-0
There are a lot of files at the top level already, personally I'd like to remove some if possible and avoid adding more.

**Comment by bjagg** [comment]
> uPortal-home recently added something similar in documenting its Committers.
@apetro may I steal your text? It's excellent, as usual.

**Comment by apetro** [comment]
@bjagg not only are you welcome to that `uPortal-home` text, it's not even stealing, and you don't need my permission. :smile: `uPortal-home` is unambiguously licensed under Apache 2, not coincidentally the same license as uPortal.

**Comment by bjagg** [comment]
Renamed to CAPS and added link in docs/README

**Pull request merged**